planet/Experimental.ocf/Shark.ocs/Scenario.txt

```diff
@@ -2,10 +2,6 @@
 Icon=1
 Title=Sharks
 
-[Player1]
-Wealth=10
-Crew=Clonk=1;
```

Comment:
You removed "Crew=Clonk=1;" from the Scenario.txt. Is that on purpose? You didn't do that for e.g. Raiders.ocs

Reply:
Yes and no. The default crew is the Clonk anyway, since the scenario is experimental I removed it here, too. For the others I did not want to mix my scenario.txt cleanup with the player proplist.